tests/FakeItEasy.Specs/StrictFakeSpecs.cs

```diff
@@ -313,6 +313,24 @@ public static void EventsManaged(IMyInterface fake, EventHandler handler)
             "And the exception message names the method"
                 .x(() => exception.Message.Should().Contain("DoIt"));
         }
+
+        [Scenario]
+        public static void NameIncludedInExpectationException(
+            IMyInterface fake,
+            Exception exception)
+        {
+            "Given a named strict fake"
+                .x(() => fake = A.Fake<IMyInterface>(o => o.Strict().Named("Foo1")));
+
+            "And I call an unconfigured method"
+                .x(() => exception = Record.Exception(() => fake.MakeIt("argument")));
+
+            "Then it should include configured name in the thrown exception"
+                .x(() => exception
+                    .Should()
+                    .BeAnExceptionOfType<ExpectationException>()
+                    .WithMessage(@"Call to unconfigured method of strict fake: FakeItEasy.Specs.StrictFakeSpecs+IMyInterface.MakeIt(name: ""argument"") on Foo1."));

```

Comment:
Perhaps this was discussed before, and I should've been paying more attention, but the format of this message doesn't sit quite well for me. Separating the "strict fake" and "on Foo1" makes it harder (for me) to figure out what I'm being told. I would've expected something more like
"Call to unconfigured method of strict fake Foo1: FakeItEasy.Specs.StrictFakeSpecs+IMyInterface.MakeIt(name: ""argument"")."

In my head, it makes it more clear what's named Foo1.

Am I alone in this?

Reply:
@blairconrad Ideally, yes, the message you suggest would be better. In terms of implementation, I'm afraid this is going to be more complex. The fake name is embedded in the call description by `DefaultFakeObjectCallFormatter`, and `StrictFakeRule` just uses that. What you suggest would require a significant refactoring...